ImP/preProcessing/testImageSimilarity.py

```diff
@@ -0,0 +1,27 @@
+import unittest 
+from preProcessor import PreProcessor 
+
+class ImageSimilarityTestCase(unittest.TestCase):
+	def setUp(self):
```

Comment:
Should have a tearDown method as well, just for ... sanitation/convention or something? Aha, I guess you'd just be setting self.processor to None.